x-pack/plugins/apm/public/components/shared/PropertiesTable/index.tsx

```diff
@@ -102,15 +102,13 @@ export function PropertiesTable({
   propKey,
   agentName
 }: {
-  propData: StringMap<any>;
+  propData?: StringMap<any>;
   propKey: string;
   agentName?: string;
 }) {
-  const hasPropData = !_.isEmpty(propData);
```

Comment:
Typescript cannot to proper flow analysis when using lodash. So instead of checking the object with `_.isEmpty`  we should simply use `obj == null`. This only works if we expect the object to be either `null` or valid.
In most cases we don't need to guard against empty objects.